egs2/siwis/tts1/local/data.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 Takenori Yoshimura
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+set -e
+set -u
+set -o pipefail
+
+log() {
+    local fname=${BASH_SOURCE[1]##*/}
+    echo -e "$(date '+%Y-%m-%dT%H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
+}
+SECONDS=0
+
+stage=-1
+stop_stage=2
+text_format=raw
+nj=8
+g2p=espeak_ng_french
+
+log "$0 $*"
+# shellcheck disable=SC1091
+. utils/parse_options.sh
+
+if [ $# -ne 0 ]; then
+    log "Error: No positional arguments are required."
+    exit 2
+fi
+
+# shellcheck disable=SC1091
+. ./path.sh || exit 1;
+# shellcheck disable=SC1091
+. ./cmd.sh || exit 1;
+# shellcheck disable=SC1091
+. ./db.sh || exit 1;
+
+if [ -z "${SIWIS}" ]; then
+    log "Fill the value of 'SIWIS' of db.sh"
+    exit 1
+fi
+
+db_root=${SIWIS}
+train_set=tr_no_dev
+dev_set=dev
+eval_set=eval1
+
+if [ ${stage} -le -1 ] && [ ${stop_stage} -ge -1 ]; then
+    log "stage -1: local/data_download.sh"
+    local/data_download.sh "${db_root}"
+fi
+
+if [ ${stage} -le 0 ] && [ ${stop_stage} -ge 0 ]; then
+    log "stage 0: local/data_prep.sh"
+    name="SiwisFrenchSpeechSynthesisDatabase"
+    local/data_prep.sh "${db_root}/${name}" "data/all" 1,2,3
+fi
```

Comment:
Just confirmation.
Is the audio trimmed well?
If not, we can add silence trimming stage.
e.g. https://github.com/espnet/espnet/blob/d628250ba5f1885aebe80e246c0791338f5a79e8/egs2/kss/tts1/local/data.sh#L72-L85

Reply:
Thank you for letting me know. No worries. The silence information is extracted from the provided HTS label file.
https://github.com/espnet/espnet/blob/00a7e457386513c7e973fe5d1e1589a487fb0054/egs2/siwis/tts1/local/data_prep.sh#L51-L54

Reply:
I miss it! Definitely better! Thanks :)